examples/splinterdb_custom_ipv4_addr_sortcmp_example.c

```diff
@@ -160,40 +149,35 @@ main()
    splinterdb_config splinterdb_cfg;
    configure_splinter_instance(&splinterdb_cfg,
                                &splinter_data_cfg,
-                               APP_DB_NAME,
-                               (APP_DEVICE_SIZE_MB * K_MiB),
-                               (APP_CACHE_SIZE_MB * K_MiB));
+                               DB_FILE_NAME,
```

Comment:
This is the only program (in this set) where configuration is wrapped into its own function.

The rest of the program is quite busy so I didn't want distracting configuration code inline'ed into main().